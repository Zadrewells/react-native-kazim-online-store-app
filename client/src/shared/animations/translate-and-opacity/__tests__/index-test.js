import React from 'react';
import renderer from 'react-test-renderer';

import {
	Text
} from 'react-native';

import {
	TranslateAndOpacityAnimation
} from 'src/shared/animations';

jest.useFakeTimers();

describe('<TranslateAndOpacity />', () => {
	const getWrapper = props => renderer.create(
		<TranslateAndOpacityAnimation
			{
			...props
			}
		>
			<Text>
				{
					'Teste'
				}
			</Text>
		</TranslateAndOpacityAnimation>
	).toJSON();

	const modelI = {
		useNativeDriver: false
	};

	it('renders correctly', () => {
		const tree = getWrapper(modelI);
		expect(tree).toMatchSnapshot();
	});
});
