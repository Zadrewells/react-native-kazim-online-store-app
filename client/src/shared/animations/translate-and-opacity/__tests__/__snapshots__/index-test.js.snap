// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TranslateAndOpacity /> renders correctly 1`] = `
<View
  style={
    Object {
      "opacity": 0,
      "transform": Array [
        Object {
          "translateX": 40,
        },
      ],
    }
  }
>
  <Text>
    Teste
  </Text>
</View>
`;
